internal/services/web/static_site_resource.go

```diff
@@ -62,6 +63,7 @@ func resourceStaticSite() *pluginsdk.Resource {
 				}, false),
 			},
 
+			// TODO 4.0 - Rename this to be `sku_name`
```

Comment:
Can you implement this change behind the `features.FourPointOhBeta()` feature flag, rather than leaving as todo?